src/app/Modules/auth/register-page/register-page.component.html

```diff
@@ -33,7 +33,7 @@
                 </button>
            </mat-form-field> 
             <div class="text-danger" *ngIf="userForm.get('password')?.hasError('required') && userForm.get('password')?.touched">
```

Comment:
The error messages of the passwords fields aren't being displayed, neither of them. You also display a message that indicates if the passwords don't match.